components/camera/client.go

```diff
@@ -89,6 +90,7 @@ func NewClientFromConn(
 	return &client{
 		ctx:                 closeCtx,
 		cancelFn:            cancelFn,
+		remoteName:          remoteName,
```

Comment:
This is how we determine if we are talking to a remote part or a main part. We need to know that so we can know whether or not we need to pop off the remote name when calling AddStream or RemoveStream